code/modules/clothing/head/helmet.dm

```diff
@@ -80,6 +77,12 @@
 	can_toggle = 1
 	toggle_cooldown = 20
 	activation_sound = 'sound/items/WEEOO1.ogg'
+	
+/obj/item/clothing/head/helmet/justice/attack_self()

```

Comment:
Well that would be better, if we were to continue with how you've proposed it in this PR, every single helmet would need to override attack_self just to play a sound effect.